scripts/load_notification_templates.py

```diff
@@ -133,3 +101,73 @@
                 template.save()
 
         logger.info(f"Written template: {template}")
+
+
+def load_sms_templates(offset):
+    logger.info("Writing SMS templates")
+
+    sms_notifications = {
+        PaymentNotificationType.SMS_INVOICE_NOTICE: "invoice_notice_{lang}.txt",
+    }
+
+    for notification_index, (notification_type, template_path) in enumerate(
+        sms_notifications.items()
+    ):
+        template = NotificationTemplate.objects.create(
+            id=notification_index + offset, type=notification_type.value,
```

Comment:
Why not auto-generated primary keys?

Reply:
To ensure the same key on all the instances where the script is run (i.e. notificaiton 1 will be the same on your local and in prod)